File: Linear Data Structures/Arrays/Practice Problems Using Arrays/C/FindMinMax.c
#include <stdio.h>

void findMaxMin(int arr[], int n) {
    int max = arr[0], min = arr[0];
    for (int i = 1; i < n; i++) {
        if (arr[i] > max) max = arr[i];
        if (arr[i] < min) min = arr[i];
    }
    printf("Max: %d, Min: %d\n", max, min);
}

int main() {
    int n;
    printf("Enter the number of elements: ");
    scanf("%d", &n);
    
    int arr[n];
    printf("Enter the elements:\n");
    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    findMaxMin(arr, n);
    return 0;
}

File: Linear Data Structures/Arrays/Practice Problems Using Arrays/Python/FindMinMax.py
def findMaxMin(arr):
    max_val = min_val = arr[0]
    for num in arr:
        if num > max_val:
            max_val = num
        if num < min_val:
            min_val = num
    print(f"Max: {max_val}, Min: {min_val}")

n = int(input("Enter the number of elements: "))
arr = [int(input(f"Enter element {i + 1}: ")) for i in range(n)]

findMaxMin(arr)
